### PR TITLE
Added fields for two-factor authentication. #6761

### PR DESCRIPTION
**Pull request by PritishC**
nereid_auth1

nereid_auth2

### CODE REVIEW

File: view/nereid_user_form2.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<data>
+    <xpath expr="/form/notebook/page[@id='permissions']" position="after">
+        <page string="Two Factor Authentication" id="twofactor">
+            <label name="use_twofactor_auth"/>
+            <field name="use_twofactor_auth"/>
+            <newline/>
```
Comment (GauravButola) [inline view/nereid_user_form2.xml]: Maybe it'll look better if both fields are in same line?

File: user.py [hunk 2]
```diff
@@ -12,3 +14,16 @@
 
 class User:
     __name__ = 'nereid.user'
+
+    use_twofactor_auth = fields.Boolean('Use Two-Factor Auth?')
```
Comment (GauravButola) [inline user.py]: Should just read `'Use Two-Factor Auth'` (no question mark) IMHO, as checkbox already implies it's a boolean field.

Comment (PritishC) [inline user.py]: Noted.

**Comment by GauravButola** [comment]
LGTM

**Pull request merged**